shell/components/ResourceDetail/Masthead.vue

```diff
@@ -516,6 +516,7 @@ export default {
             <button
               v-if="isView"
               ref="actions"
+              data-testid="mathead-action-menu"
```

Comment:
hmm.. I'm just noticing a typo in the id for this backport

```suggestion
              data-testid="masthead-action-menu"
```

This is also present in the `master` branch, so we can keep it as-is if that's more straightforward for the backport.